docker/zap-baseline.py

```diff
@@ -321,10 +321,23 @@ def main(argv):
         time.sleep(2)
 
         # Spider target
-        zap_spider(zap, target)
+        '''
+        Here the zap spider is supposed to only use a context if a context file is given
```

Comment:
I'd suggest removing this comment, it's repeating what the code is doing (same for the others).